### PR TITLE
Add windows server core instructions

### PR DESCRIPTION
**Pull request by RassK**
## What

Adds Windows Server Core Docker image setup instructions

### CODE REVIEW

File: docs/internal/docker-windows.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Setting up SignalFx Instrumentation for .NET to run in Windows Nano Server Docker image.
+
+```Dockerfile
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-nanoserver-ltsc2022
+
+# Setup SignalFx environment variables
+ENV CORECLR_ENABLE_PROFILING=1 \
+    CORECLR_PROFILER='{B4C89B0F-9908-4F73-9F59-0D77C5A06874}' \
+    CORECLR_PROFILER_PATH_32='C:/signalfx/win-x86/SignalFx.Tracing.ClrProfiler.Native.dll' \
```
Comment (pjanotti) [inline docs/internal/docker-windows.md:11]: Because this env var relies on the download and unzip below it seems better to have it after those steps.

**Review by pjanotti — COMMENTED** [review]
Two suggestions @RassK

File: docs/internal/docker-windows.md (hunk 1 above)
Comment (RassK) [reply docs/internal/docker-windows.md:11]: @pjanotti do you mean to move entire block or separate only single env?

Comment (RassK) [reply docs/internal/docker-windows.md]: Updated.

File: docs/internal/docker-windows.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Setup SignalFx environment variables
+ENV CORECLR_ENABLE_PROFILING=1 \
+    CORECLR_PROFILER='{B4C89B0F-9908-4F73-9F59-0D77C5A06874}' \
+    SIGNALFX_ENDPOINT_URL='<my-endpoint-here>' \
+
+ARG TRACER_VERSION=0.2.4
+
+# Download and install SignalFx Instrumentation for .NET MSI
+RUN New-Item -Path 'C:\Temp' -ItemType Directory
+RUN Invoke-WebRequest "https://github.com/signalfx/signalfx-dotnet-tracing/releases/download/v${env:TRACER_VERSION}/signalfx-dotnet-tracing-${env:TRACER_VERSION}-x64.msi" -OutFile "C:\Temp\signalfx-dotnet-tracing.msi"
```
[the remark this answers is not in the document]
Comment (RassK) [reply docs/internal/docker-windows.md:56]: Updated.

**Comment by pjanotti** [comment]
It seems some flaky runs https://github.com/signalfx/signalfx-dotnet-tracing/runs/6829363518?check_suite_focus=true#step:4:1260

Re-kicking failed ones.

**Pull request merged**